client/odysseus/message/index.tsx

```diff
@@ -0,0 +1,74 @@
+import classnames from 'classnames';
+import { RefObject, useState } from 'react';
+import AsyncLoad from 'calypso/components/async-load';
+import type { Message } from '../types';
+
+import './style.scss';
+
+type ChatMessageProps = {
+	message: Message;
+	isLast: boolean;
+	messageEndRef: RefObject< HTMLDivElement >;
+};
+
+const ChatMessage = ( { message, isLast, messageEndRef }: ChatMessageProps ) => {
+	const [ liked, setLiked ] = useState( false );
+	const [ disliked, setDisliked ] = useState( false );
+
+	const isUser = message.role === 'user';
+
+	const onLike = () => {
+		if ( liked ) {
+			setLiked( false );
+			setDisliked( false );
+			return;
+		}
+		setLiked( true );
+		setDisliked( false );
+	};
+
+	const onDislike = () => {
+		if ( disliked ) {
+			setLiked( false );
+			setDisliked( false );
+			return;
+		}
+		setLiked( false );
+		setDisliked( true );
+	};
+
+	return (
+		<div
+			ref={ isLast ? messageEndRef : null }
+			className={ `chatbox-message ${ isUser ? 'user' : 'wapuu' }` }
+		>
+			<AsyncLoad require="react-markdown" children={ message.content } />
+			{ ! isUser && (
+				<div className="message-actions">
+					<button
+						aria-label="Like this message"
```

Comment:
Did we mean to localize this string?

Reply:
We are not going to localise anything yet, as everything changing quite often and we don't want extra work for i18n team :)

Reply:
Just keep in mind that those are easily forgotten once we push to production. Might be worth to add a TODO comment or a TODO item somewhere 😉

Reply:
Added to our board :) 

https://github.com/orgs/Automattic/projects/708/views/1?pane=issue&itemId=33085119